pages/common/roave-backward-compatibility-check.md

```diff
@@ -0,0 +1,20 @@
+# roave-backward-compatibility-check
+
+> A tool that can be used to verify BC breaks between two versions of a PHP library.
```

Comment:
I would prefer to have "backward compatibility breaks" istead of "BC breaks" as it looks easier to understand at first sight.